README.md

```diff
@@ -1,17 +1,19 @@
 [![Actions Status](https://github.com/libass/JavascriptSubtitlesOctopus/workflows/Emscripten/badge.svg)](https://github.com/libass/JavascriptSubtitlesOctopus/actions)
 
-SubtitlesOctopus displays subtitles in .ass format and easily integrates with HTML5 videos. It supports all SSA/ASS features and fully compatible with [libass](https://github.com/libass/libass).
+SubtitlesOctopus displays subtitles in .ass format and easily integrates with HTML5 videos.
+Since it utilises [libass](https://github.com/libass/libass), SubtitlesOctopus supports most SSA/ASS features
```

Comment:
Very minor and arguable nit: “utilises” is a slightly unusual and flavoured word; I’d just write “uses”, which is more common and neutral.

I couldn’t put my finger on it myself, but my dictionary says:
> make practical and effective use of:
> > <i>vitamin C helps your body utilize the iron present in your diet</i>

Reply:
Changed to `uses` and added libass to included libraries, thanks!